mod_reforged/hooks/entity/tactical/enemies/spider.nut

```diff
@@ -76,6 +76,7 @@
 		this.m.Skills.add(this.new("scripts/skills/racial/spider_racial"));
 
 	    // Reforged
+	    b.DamageDirectAdd += 0.10;
```

Comment:
I feel Between the Ribs will be more of a buff than this honestly. +10% damage and armor pen per spider ally vs. just +10% armor pen once.

Reply:
Let's try Between the Ribs. Spiders do feel quite weak right now. They die easily, so this will make them more dangerous glass cannons - and you can "avoid" the damage bonus by killing them strategically to remove adjacency bonuses.

Reply:
resolved by removing 10% direct damage add and instead adding between the ribs